test/membrane_rtc_engine/engine_test.exs

```diff
@@ -329,103 +310,6 @@ defmodule Membrane.RTC.EngineTest do
     end
   end
 
-  describe "Engine linking test" do
-    @tag :skip
-    test "engine properly links for both formats", %{rtc_engine: rtc_engine} do
```

Comment:
I am not sure whether we should delete this test but after removing :raw format I feel like it is useless as we have a bunch of integration tests :thinking:

Reply:
Yeah, seems redudant